### DISCUSSION TITLE
Journey player on classic world

### DISCUSSION THREAD

**Discussion by eliximin** [Pryaxis/TShock#2959] [Q&A]
How do I let journey characters join a classic world? I tried the Journey Mixer plugin, but I'm new to GitHub so I didn't know how to install it correctly.

**Comment by sgkoishi** [comment]
[TShock documentation](https://ikebukuro.tshock.co/):
> To install more plugins, add them to the `ServerPlugins` folder.

Also, TShock is 5.x right now and many plugins from 4.x might _just not working_; in that case probably you need to compile it yourself or find other versions ¯\_(ツ)_/¯

**Reply by eliximin** [reply]
im horrible with code

**Comment by ghost** [comment]
There is a Tshock config option to set that, i know it could be annoying to find it but try you best!

**Comment by Suzukax12345** [comment]
any update on where to find this in tshock config

**Reply by zyrafaq** [reply]
> any update on where to find this in tshock config

First make sure the [softcore only](https://github.com/Pryaxis/TShock/wiki/config-file-descriptions#softcoreonly) config option is disabled, then install [Crossplay](https://github.com/Moneylover3246/Crossplay) and enable `support_journey_clients`. Hope this helps
